Materials/explainer.md

```diff
@@ -0,0 +1,128 @@
+# Materials in Web Applications
+
+Authors: [Diego Gonzalez](https://github.com/diekus), [Andy Luhrs](https://github.com/aluhrs13)
+
+## Status of this Document
+
+This document is a starting point for engaging the community and standards bodies in developing collaborative solutions fit for standardization. As the solutions to problems described in this document progress along the standards-track, we will retain this document as an archive and use this section to keep the community up-to-date with the most current standards venue and content location of future work and discussions.
+
+- This document status: **Brainstorming**
+- Expected venue: TBD
+- **Current version: this document**
+
+## Introduction
+
+*Materials* are visual effects applied to UX surfaces that resemble real life artifacts. Modern applications can use *materials* to create a unified and integrated appearance with the underlying OS. Materials are a powerful way to:
+- create stronger contextual relationship between the content the user is interacting an related UX (like context menus or side bars).
+- indicate window (active/inactive) focus state.
+- establish visual hierarchy.
+- communicate separation between layers to help a user retain a sense of space.
+
+[Apple OSs](https://developer.apple.com/design/human-interface-guidelines/materials) have several materials, ranging from `ultraThin` to `ultraThick`.
+
+![Stock app on macOS displaying a left panel with a translucent material background](macMaterial.png)
+
+[Windows 11](https://learn.microsoft.com/en-us/windows/apps/design/signature-experiences/materials) also has a similar concept, with `Mica`, `Mica Alt` and `Acrylic`.
+
+![Copilot app on Windows 11 displaying a left panel with a translucent material background](winMaterial.png)
+
+This explainer addresses allowing web applications to use materials that might be available in the underlying platform. When enabled, and if the platform supports it, the frame of the installed web application will use the specified visual effect, and if the background of the page is transparent the material would be visible.
+
+> **Note:** A developer can already create and mimic experiences based on opaque surfaces. By using images and paterns there is a high chance that a desired platform aesthetic can be achieved (like _brushed-metal_). With newer and more powerful hardware, platforms are starting to add transparency effects to their UX surfaces. The intention of this API is to allow a web application's frame to use materials with transparency effects like the ones in the screenshots below.
+
+## Goals
+
+- Enable the frame of an installed web application to use OS materials.
+- Design a solution that will scale to future design systems.
```

Comment:
```suggestion
- Design a solution that will scale to future OS materials.
```

(design system has a very specific meaning, which doesn't match what you're trying to say here).